### DISCUSSION TITLE
表格套表格的有实现嘛，示例有没有

### DISCUSSION THREAD

**Discussion by wsxe9988** [alibaba/formily#1969] [Q&A 2.x] [answered]
表格套表格的有实现嘛，示例有没有
类似于下图
![image](https://user-images.githubusercontent.com/5828057/128653668-df9c7095-0458-4d7b-9f41-9067e426fac7.png)

**Comment by janryWang** [comment, marked as answer]
自己写个自定义组件，代码例子直接看@formily/antd的源码
